interface/interface.dm

```diff
@@ -256,17 +256,9 @@ Any-Mode: (hotkey doesn't need to be on)
 	set name = "map"
 	set desc = "View the current map in the webviewer"
 	set hidden = 1
```

Comment:
Actually, why is the verb hidden?

Reply:
I'm not really sure, that's how I found it. Do you want me to change that?

Reply:
It's hidden because it was originally tied to a UI element that no longer exists. I see no real reason for it to be hidden.